File: app/src/main/java/com/philsoft/metrotripper/utils/ui/SimplePanelSlideListener.java
package com.philsoft.metrotripper.utils.ui;

import android.view.View;

import com.sothree.slidinguppanel.SlidingUpPanelLayout;

public class SimplePanelSlideListener implements SlidingUpPanelLayout.PanelSlideListener {

    @Override
    public void onPanelSlide(View panel, float slideOffset) {
        // Can be overridden
    }

    @Override
    public void onPanelStateChanged(View panel, SlidingUpPanelLayout.PanelState previousState, SlidingUpPanelLayout.PanelState newState) {

    }
}

File: app/src/main/java/com/philsoft/metrotripper/utils/ui/SimplePanelSlideListener.kt
package com.philsoft.metrotripper.utils.ui

import android.view.View

import com.sothree.slidinguppanel.SlidingUpPanelLayout

open class SimplePanelSlideListener : SlidingUpPanelLayout.PanelSlideListener {

    override fun onPanelSlide(panel: View, slideOffset: Float) {
        // Can be overridden
    }

    override fun onPanelStateChanged(panel: View, previousState: SlidingUpPanelLayout.PanelState, newState: SlidingUpPanelLayout.PanelState) {

    }
}
